### ISSUE TITLE
Blurry TMail web interface

### ISSUE THREAD

**Issue by Facyla** [linagora/tmail-flutter#2618]
### Description
On Firefox desktop, the Twake Mail interface texts look "blurry", as if a very thin "text-shadow" was applied to the text. No flickering, but this makes the reading difficult and unpleasant for the eye.
Note: it doesn't appear when writing a new email, nor on popins.
![image](https://github.com/linagora/tmail-flutter/assets/286492/245e7a21-399f-44d7-8574-affd75f21c2f)

### Expected behaviour
sharp texts.

### Cause
I suspect this is linked with the use of "canvas" by Flutter for rendering, which renders the text as if it were an image.

### Identified solution
https://www.geeksforgeeks.org/how-to-sharpen-blurry-text-in-html5-canvas/

**Comment by dab246** [comment]
Does this happen on other browsers?

**Comment by chibenwa** [comment]
I remember @fabienmoyon  complaining about this...

**Comment by fabienmoyon** [comment]
I think this happen when the window zoom for Twake Mail is lower or higher of the default value : 100%.

Is it also your configuration case @Facyla ?

**Comment by Facyla** [comment]
Nope, I use various zoom configurations, but mostly 100%, and when testing i am used to switch and check various situations. In that case it happened whatever the zoom level - it is rather related to the screen size and resolution, and how canvas handles this. 

Anyway, the way to fix the issue is detailed in the link i attached => can be solved by using [window.devicepixelratio property](https://www.geeksforgeeks.org/html-dom-window-devicepixelratio-property/)

**Comment by chibenwa** [comment]
Is this still happening?
